js-exercises/freq-sort/freqSort.js

```diff
@@ -0,0 +1,17 @@
+function freqSort(arrayToSort) {
+  let arrayElementFrequency = {};
+  arrayToSort.forEach((element) => {
+    element = element.normalize();
+
+    if (arrayElementFrequency[element]) arrayElementFrequency[element] += 1;
+    else arrayElementFrequency[element] = 1;
+  });
```

Comment:
- Try to put this frequency management related in a separate function. 
- Also, try to put extra check such as 'arrayToSort' contains an array or not by using Array.isArrray method.